versioned_docs/version-24.10/graph-views/map-legacy-eol.md

```diff
@@ -0,0 +1,77 @@
+---
+id: map-legacy-eol
+title: MAP Legacy end of life
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> **MAP Legacy is no longer available from version 24.10, you have to use MAP.**
+
+This topic describes the steps to follow regarding MAP Legacy once you have migrated to the available version of MAP.
+
+## How to fully uninstall MAP Legacy
+
+As MAP Legacy is no longer delivered, you must uninstall it completely following the instructions below.
+
+> It is mandatory to perform these steps now, as you may encounter issues when upgrading to later versions.
+
+### Migrate to Centreon MAP
+
+> MAP Legacy must be uninstalled after you have migrated to MAP.
```

Comment:
I would remove this one as this feels like too much repetition when reading the page.